### ISSUE TITLE
duniverse sticks on a pin

### ISSUE THREAD

**Issue by dinosaure** [avsm/duniverse#58]
When I tried to use duniverse with mirage+dune (https://github.com/TheLortex/mirage/tree/full-dune), duniverse is stick on the pin of `mirage-unikernel-pasteur-unix`. Currently the work is done on `pasteur` (https://github.com/dinosaure/pasteur/tree/duniverse).

**Comment by dinosaure** [comment]
Ok, a infinite loop appears because an `opam pin` ask if we want to edit the OPAM file mostly because `mirage-unikernel-pasteur-unix` is not in the worktree - I believe `duniverse` does an `opam pin` which, by default, default the Git version. However, the mirage tool generate the OPAM file but did not add it as a new element of the git repository.

The way to fix it is:
```sh
$ mirage configure -t unix
$ git add *.opam && git commit -m .
$ duniverse init
```

**Comment by NathanReb** [comment]
Thanks for reporting and debugging this! Other users, myself included, have been complaining about issues with non version controlled opam files.

I'll look into how we can fix that!

**Comment by NathanReb** [comment]
I believe this was fixed by #63 so I'm closing the issue but feel free to re-open if the symptoms re-appears!
